spec/factories/goals.rb

```diff
@@ -1,15 +1,15 @@
 FactoryGirl.define do
   factory :goal do
-    user
+    # user
```

Comment:
I think these commented out parts should be commented in, since the feedbackfruits-api requires these functionalitites for activities as well